Translated the new dialogs into Dutch

pmjherman opened the pull request:
I have translated the new dialogs into Dutch in the file seamly2d_nl_NL.ts. I have tested this file in my local build and everything seemed OK. 
This is my very first pull request. Excuses if something goes wrong!

pmjherman committed:
Translated the new dialogs into Dutch

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/FashionFreedom/Seamly2D?pullRequest=854) <br/>All committers have signed the CLA.

DSCaskey commented:
I will check it out later tonight.

DSCaskey commented:
@pmjherman 

Just a note... create an issue with which we can link the PR to.

pmjherman committed:
Lupdate run on seamly2D

Lupdate run on seamly2D, not on measurements

pmjherman committed:
Merge pull request #1 from pmjherman/NLtranslation

Lupdate run on seamly2D

pmjherman committed:
Merge branch 'develop' of https://github.com/pmjherman/Seamly2D into develop

pmjherman commented:
I have run Lupdate, checked that all my translations were still there. All other languages have also been 'updated', without changes however.
I have built the program on Linux and successfully run all the tests. I hope a PR could now be successful, otherwise I would not know what to change.

DSCaskey commented:
@pmjherman 

>All other languages have also been 'updated', without changes however.

That's ok. As it should be. 

 Running the workflow now, and the appimage build and Linux tests failed... of course due to the fact the workflow is on your fork,  not the main.   The Mac and Windows passes. 

> I hope a PR could now be successful, otherwise I would not know what to change.

One of 3 things... either Susan merges the PR as is OR she gives you permission to the main where you can create a new  branch and  PR there (where the Linux is likely to be  successful) OR someone else with permission pulls your changes and then pushes and creates a PR on the main.

csett86 commented:
@pmjherman the tests still seem to be failing. You can run them locally with 

     make check

csett86 reviewed (changes requested):
Please fix the tests first, then I can merge it

pmjherman committed:
corrected translation files causing make check failures

pmjherman commented:
My local make check failed on four punctuation flaws in my translations (missing dot at end, or missing leading space). I corrected these, but then the tests also failed on some of the existing translation files for measurements. I do not know why, since I had not touched these files. I corrected them anyway, plus I ran Lupdate on all the measurement translations too. My local build now has no failures left in make check. I cannot say I have inspected all of the 1 million (!) lines of output, but a search for FAIL did not deliver anything, and I checked that all of the tests had "0 failed".

csett86 commented:
Thanks a lot, I have also tested it locally under Linux and Mac, and it looks great now. Thanks a lot for your great contribution with this massive amount of translation!

I will wait for the CI to finish (the last failure was just a github internal error, nothing with your code), then I'll merge this :)
